### DISCUSSION TITLE
YAML file showing errors, tried single/double spaces after colons & before colons

### DISCUSSION THREAD

**Discussion by pjuned** [learndevopsonline/learndevopsonline#1590] [Q&A] [answered]
YAML file showing errors, tried single/double spaces after colons & before colons :
![yml1](https://github.com/learndevopsonline/learndevopsonline/assets/148682494/b9e7bfbb-c8a6-418b-83ac-b3e6c8e4a20b)
![yml2](https://github.com/learndevopsonline/learndevopsonline/assets/148682494/3b89b91d-2b22-44b3-932d-b00cf5494b94)

**Comment by sreedharm07** [comment, marked as answer]
hi @pjuned 
it is expecting ( : ) after **ping:**
please give it.
![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/73e8bc11-6c84-49ac-81fd-9c2b25c830f7)
